spec/stat_generator_spec.rb

```diff
@@ -24,6 +24,66 @@
     end
   end
 
+  describe '#checks highest total score and lowest total score' do
+    it 'Calculates the highest total score for all games' do
+      expect(@stat_generator.highest_total_score).to eq(11)
+      # @game2 = Game.new() try to prove that 
+      expect(@stat_generator.highest_total_score).to_not eq(1)
```

Comment:
We'll need to fix these tests like we talked about. Right now, this is just showing that the method returns the integer value of 11 and possibly that's all it does. The second expect is only showing that it's not 1. So for all we know, the method's only purpose is still to just return the value of 11. 

A lot of my tests are similar at the moment, but we can go over them later to make them more thorough. Possibly after learning about mock and stubbing.